_locales/en/messages.json

```diff
@@ -389,5 +389,53 @@
     },
     "backup_file_info": {
         "message": "Backup your data to a file."
+    },
+    "advisor": {
+        "message": "Advisor"
+    },
+    "advisor_insight_password_not_set": {
+        "message": "Password should be set to protect your data."
+    },
+    "advisor_insight_auto_lock_not_set": {
+        "message": "Auto lock should be set to protect your data."
+    },
+    "advisor_insight_browser_sync_not_enabled": {
+        "message": "Browser sync should be enabled to sync data across browsers."
+    },
+    "advisor_insight_zoom_not_set_normal": {
+        "message": "Zoom should be set to 100% unless you expect scaling."
+    },
+    "advisor_insight_3rd_party_storage_not_enabled": {
+        "message": "3rd party storage services should be enabled to backup data."
```

Comment:
Removed this insight. Will add a new field in local storage to record last time to make a backup, so that we can give a warning if the user doesn't backup data for a long time.